src/app/features/ui/side-menu/side-menu.component.scss

```diff
@@ -0,0 +1,131 @@
+@use './src/styles/colors' as colors;
+@use './src/styles/breakpoints' as breakpoints;
+@use './src/styles/mixins' as mixins;
+
+.side-menu-container {
+  height: calc(100vh - 52px);
+  display: flex;
+  position: fixed;
+  z-index: 999;
+}
+
+.side-menu-container__left {
+  @include mixins.gradient(0deg, colors.$bg-gradient-primary-1);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  width: clamp(70px, 12vw, 100px);
+  padding-bottom: 0;
+  padding-top: clamp(0px, 10vh, 100px);
+  padding-bottom: clamp(0px, 5vh, 100px);
+  height: 100%;
+
+  &Top {
+    width: 100%;
+  }
+
+  &Bottom {
```

Comment:
co to za dziwne nazwy klas `__leftTop`, `__leftBottom` ? czemu nie `__left-top` ?

Reply:
Nazwy klas zostały poprawione.